hpi-schul-cloud/schulcloud-server
# BC-6965 - Fix videoconference guest label bug

# Description

QA note:change tested on env: 
https://bc-6965-fix-videoconference-guest.dbc.dbildungscloud.dev/
https://bc-6965-fix-videoconference-guest.nbc.dbildungscloud.dev/
https://bc-6965-fix-videoconference-guest.brb.dbildungscloud.dev/
No issues found. Tested videoconf on course and team. see ticket: 
https://ticketsystem.dbildungscloud.de/browse/BC-6965



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-6965


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
